src/Neo.CLI/CLI/MainService.Logger.cs

```diff
@@ -123,23 +124,21 @@ void ILoggingHandler.Utility_Logging_Handler(string source, LogLevel level, obje
                     currentColor.Apply();
                     Console.WriteLine();
                 }
-
-                if (string.IsNullOrEmpty(Settings.Default.Logger.Path)) return;
-                var sb = new StringBuilder(source);
-                foreach (var c in GetInvalidFileNameChars())
-                    sb.Replace(c, '-');
-                var path = Combine(Settings.Default.Logger.Path, sb.ToString());
-                Directory.CreateDirectory(path);
-                path = Combine(path, $"{now:yyyy-MM-dd}.log");
-                try
-                {
-                    File.AppendAllLines(path, new[] { $"[{level}]{log} {message}" });
-                }
-                catch (IOException)
-                {
-                    Console.WriteLine("Error writing the log file: " + path);
-                }
             }
+
+            if (string.IsNullOrEmpty(Settings.Default.Logger.Path)) return;
+
+            var logEventLevel = level switch
+            {
+                LogLevel.Debug => LogEventLevel.Debug,
+                LogLevel.Info => LogEventLevel.Information,
+                LogLevel.Warning => LogEventLevel.Warning,
+                LogLevel.Error => LogEventLevel.Error,
+                LogLevel.Fatal => LogEventLevel.Fatal,
+                _ => LogEventLevel.Information,
+            };
+
+            Log.GetLogger(source).Write(logEventLevel, "{message}", message);
```

Comment:
```suggestion
            Log.GetLogger(source).Write(logEventLevel, message, null);
```
It won't be faster without any replace?